editor/editor_inspector.cpp

```diff
@@ -915,14 +927,20 @@ static Control *make_help_bit(const String &p_text, bool p_property) {
 	}
 	text += "[u][b]" + property_name + "[/b][/u]";
 
-	if (slices.size() > 1) {
+	if (slices.size() > 1 && slices[1] != "") {
 		String property_doc = slices[1].strip_edges();
 		if (property_name != property_doc) {
 			text += "\n" + property_doc;
 		}
 	} else {
 		text += "\n[i]" + TTR("No description.") + "[/i]";
 	}
+
+	// Add inactive warning
+	if (slices.size() > 2 && slices[2] != "") {
+		text += "\n[b][color=yellow]" + TTR("Warning: this property does not have an effect as it requires ") + slices[2].strip_edges() + "[/color][/b]";
```

Comment:
This is a bit awkward, because it forces a specific structure from the dependency string.
Maybe something like
```
Warning: This property does not have effect, because it doesn't meet dependency: %s.
```
would be better, where `%s` is e.g. `some_property must be enabled.`.